data/data-pipeline/data_pipeline/tests/sources/example/test_etl.py

```diff
@@ -86,7 +87,23 @@ def setup_method(self, _method, filename=__file__):
         self._DATA_DIRECTORY_FOR_TEST = pathlib.Path(filename).parent / "data"
 
     def _get_instance_of_etl_class(self) -> Type[ExtractTransformLoad]:
-        return self._ETL_CLASS()
+        etl_class = self._ETL_CLASS()
+
+        # Find out what unique state codes are present in the test fixture data.
+        states_expected_from_fixtures = {
+            x[0:2] for x in self._FIXTURES_SHARED_TRACT_IDS
+        }
+
+        # Set values to match test fixtures
+        etl_class.EXPECTED_MISSING_STATES = [
```

Comment:
Oh, this is actually because `etl_class.EXPECTED_MISSING_STATES` in the test is going to need to be the *opposite* of `states_expected_from_fixtures`. `states_expected_from_fixtures` will be just two states `['06','15']` and etl_class.EXPECTED_MISSING_STATES is going to be every single FIPS code except those two.